template/test.jinja

```diff
@@ -2,3 +2,8 @@
 
 RUN cd /test && \
     python3 serial-cpu.py
+
+{{ RUN }} git clone --recursive --branch {{ COXETER_VERSION }} --depth 1 https://github.com/glotzerlab/coxeter \
```

Comment:
```suggestion
RUN git clone --recursive --branch {{ COXETER_VERSION }} --depth 1 https://github.com/glotzerlab/coxeter \
```
Sorry about more bad advice. `test.jinja` doesn't define the RUN variable, so just use `RUN` directly here.

Reply:
gotcha, that makes sense!